doc/topics/development/salt_extensions.rst

```diff
@@ -40,31 +25,45 @@ The criteria to determine which category to place modules in will follow these r
 
   a. Modules that function across multiple operating systems like cmd and file.
 
-## Supported Modules
+Supported Modules
+-----------------
+
+Supported modules will be moved to their own repositories within the `SaltStack Github`_
+organization where they can be maintained separately from the Salt codebase.
 
 1. Modules to support specific operating systems traditionally maintained by the core team
-– such as RedHat, MacOS, Windows, Solaris, etc.
+   – such as RedHat, MacOS, Windows, Solaris, etc.
 
 2. Modules to support specific but critical applications, such as Apache, MySQL, etc.
 
 3. Modules created and maintained as part of VMware backed support agreements and
-contracts.
+   contracts.
+
+Community Modules
+-----------------
+
+Remaining modules will be deprecated from the Salt Core codebase and community members
+will be able to continue independent maintainership if they are interested. Some plugins are
+almost exclusively maintained by external corporations – if these corporations wish for formal
+documentation outlining transfer of ownership it can be handled on a case-by-case basis. The
+community modules can be hosted either in individual or corporate source control systems,
+alternatively they can also be hosted in the community run `Salt Extensions`_ Github organization,
+that will operate like the `Salt Formulas`_ Github organization.
```

Comment:
A few suggestions for this paragraph:

- Can we add a "the" in front of "remaining modules"?
- Just to reduce sentence complexity, I'd break up the first sentence into two sentences.
- Instead of "continue independent maintainership," how about instead: "Community members can volunteer to maintain these modules if they are interested."
- Instead of "external corporations - if these...", let's just make them two sentences. 

Question: I'm confused around the transfer of ownership statement. Transfer to whom? From Salt to the company?

I was also wondering if we should answer the question about what are the responsibilities of the community members who maintain the extensions and how they can transfer ownership if they no longer want to maintain them down the road.

Reply:
... source control systems. Alternatively...